### PR TITLE
Updated event API endpoints to exclude past events

### PR DESCRIPTION
**Pull request by kevhuang**
Will add the ability for organizers to end events in a separate pull request

### CODE REVIEW

File: server/api/event/index.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+// Return a list of all active events
 router.get('/', function(req, res) {
- db.Event.findAll()
-   .then(function(results){
-     res.json(results);
-   });
+ db.Event.findAll({
+  where: {
+    end: {
+      $gt: new Date() // Filters events where end date is greater than the current timestamp
```
Comment (ekong2) [inline server/api/event/index.js]: try using new Date(new Date().getTime() + 60 * 60 * 24 * 1000)

when we save to the database, we use this format

Comment (ekong2) [inline server/api/event/index.js:9]: nvrm- date format is correct

**Comment by ekong2** [comment]
:100:

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**